src/i18n/RuleEngine.js

```diff
@@ -638,6 +638,34 @@ export default {
     zh: "例如：${'{'}payload{'}'}, ${'{'}clientid{'}'}, ${'{'}topic{'}'}, ${'{'}username{'}'} 等。请根据使用数据桥接的业务需求来选择字段，置空则原样转发消息。",
     en: "For example: ${'{'}payload{'}'}, ${'{'}clientid{'}'}, ${'{'}topic{'}'} , ${'{'}username{'}'}, etc. Use fields according to the data bridges requirements of your business and forwards the message as it is if it is empty.",
   },
+  pubProp: {
+    zh: '发布属性',
+    en: 'Publish Properties',
+  },
+  payloadFormatIndicator: {
+    zh: '有效载荷指示器',
+    en: 'Payload Format Indicator',
+  },
+  messageExpiryInterval: {
+    zh: '遗嘱消息过期时间',
+    en: 'Message Expiry Interval',
+  },
+  contentType: {
+    zh: '遗嘱消息描述',
+    en: 'Content Type',
+  },
```

Comment:
It seems this has nothing to do with the last will message; I believe you might have referred to the wrong translation.

Reply:
oh, let me see